packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/plan-language-tags-guesser.ts

```diff
@@ -141,7 +144,8 @@ export const getLanguageTagFromPlanMetadata = async (
       return handleEnglishPlan(plan);
     }
 
-    let lang = findLocaleInTitle(detectionResult[0].language, plan.nickname!);
+    if (!plan.nickname) throw new Error('Plan nickname is required');
+    let lang = findLocaleInTitle(detectionResult[0].language, plan.nickname);
```

Comment:
I would ask the same thing here to pass the `plan.id` in all these new Errors to Sentry (`Sentry.withScope...`); though it's less important here to pass it along in Sentry, since this is part of a temporary migration script for FXA-4089.

Reply:
See https://github.com/mozilla/fxa/pull/14508#issuecomment-1335470569